### PR TITLE
fix BUG #1398,fix bsh,xml-resolver default version cannot access

### PR DESCRIPTION
**Pull request by zhengchaoken**
## Purpose of this pull request
bsh,xml-resolver 的默认版本无法直接访问到，更新版本到可访问版本。本地maven 打包测试通过。

## Which issue you fix
Fixes # (1398).

## Checklist:

- [x] I have executed the **'mvn spotless:apply'** command to format my code.
- [x] I have a meaningful commit message (including the issue id, **the template of commit message is '[label-type-#issue-id][fixed-module] a meaningful commit message.'**)
- [x] I have performed a self-review of my own code.
- [] I have commented my code, particularly in hard-to-understand areas.
- [] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have checked my code and corrected any misspellings.
- [x] My commit is only one. (If there are multiple commits, you can use 'git squash' to compress multiple commits into one.)

### CODE REVIEW

File: chunjun-assembly/pom.xml [hunk 1]
```diff
@@ -33,15 +33,50 @@
 	<packaging>pom</packaging>
 	<name>ChunJun : Assembly</name>
 
-
 	<properties>
 		<skipTests>true</skipTests>
+		<!--version same as flink-1.12.7-bin-scala_2.12.tgz/lib/log4j-* -->
+		<log4j2.version>2.16.0</log4j2.version>
```
Comment (zoudaokoulife) [inline chunjun-assembly/pom.xml:39]: version 2.17.0 may be better for it  fix CVE-2021-45105

Comment (zhengchaoken) [reply chunjun-assembly/pom.xml:39]: yes , 2.16.0 is the version flink 1.12.7 used , just same as it .

we(chunjun)  can update the version alone .

**Review by FlechazoW — APPROVED** [review]
LGTM

**Review by PaddyMelody — APPROVED** [review]
+1

**Pull request merged**